File: Feitos em C++ Parte 2/1858.cpp
#include <bits/stdc++.h>
using namespace std;
int main(){
    int ent,aux,pega;
    cin>>ent;
    int vet[ent];
    for(int x=0;x<ent;x++){
        cin>>vet[x];
        if(x==0){
            aux=vet[x];
            pega=x+1;
        }
        else if(aux>vet[x]){
            aux=vet[x];
            pega=x+1;
        }
    }
    cout<<pega<<endl;
    return 0;
}

File: Feitos em Java/1858.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner entrada = new Scanner(System.in);
        int ent=entrada.nextInt();
        int aux=0,pega=0;
        int [] vet=new int [ent];
        for(int x=0;x<ent;x++){
            vet[x]=entrada.nextInt();
            if(x==0){
                aux=vet[x];
                pega=x+1;
            }
            else if(aux>vet[x]){
                aux=vet[x];
                pega=x+1;
            }
        }
        System.out.printf("%d\n",pega);
    }
}